7.3/Dockerfile.dev

```diff
@@ -0,0 +1,36 @@
+FROM jorge07/alpine-php:7.3
+
+ARG USER=root
+ARG PASSWORD=root
+
+ARG COMPOSER_VERSION=1.6.3
+
+RUN apk add -U --no-cache \
+        php7-pear@cast \
+        openssh \
+        supervisor \
+        autoconf \
+        git \
+        curl \
+        wget \
+        make \
+        zip \
+        php7-xdebug@cast \
```

Comment:
Is xdebug package released for 7.3?

Reply:
https://github.com/codecasts/php-alpine/blob/master/scripts/v3.8/php-7.3/php7-xdebug/APKBUILD

It seems. I'll try to re-run this today

Reply:
👏

Reply:
https://github.com/codecasts/php-alpine/issues/53

Reply:
🤷‍♂️

Reply:
https://www.patreon.com/posts/php-7-3-support-23353264